src/Components/SideBar/SideBar.jsx

```diff
@@ -125,7 +125,7 @@ const SideBar = () => {
             />
           ))
         ) : (
-          <div>There are no records of articles</div>
+          <div>Loading articles...</div>
```

Comment:
Acutally, now that I think of it more, we need to consider the different states the side bar could be in.

It could be loading articles or there could just be no articles. We should update the sidebar to be able to reflect both.

Reply:
Ah, I see. I will make that change 👍